src/sync.rs

```diff
@@ -0,0 +1,41 @@
+pub fn sync() {
+    let (_fpm_config, base_dir) = fpm::check();
+
+    std::fs::create_dir_all(format!("{}/.history", base_dir.as_str()).as_str())
+        .expect("failed to create build folder");
+
+    fpm::process_dir(
+        base_dir.clone(),
+        0,
+        base_dir,
+        &[".history", ".build"],
+        &write,
+    );
+}
+
+fn write(id: &str, doc: String, base_path: String, _depth: usize) {
+    use chrono::prelude::*;
+    use std::io::Write;
+
+    if id.starts_with(".history") {
+        return;
+    }
+
+    if id.contains('/') {
+        let (dir, _) = id.rsplit_once('/').unwrap();
+        std::fs::create_dir_all(format!("{}/.history/{}", base_path.as_str(), dir))
+            .expect("failed to create directory folder for doc");
+    }
+
+    let new_file_path = format!(
+        "{}/.history/{}",
+        base_path.as_str(),
+        id.replace(".ftd", &format!(".{}.ftd", Utc::now().timestamp()))
```

Comment:
```rust
use std::time::SystemTime;

match SystemTime::now().duration_since(SystemTime::UNIX_EPOCH) {
    Ok(n) => println!("1970-01-01 00:00:00 UTC was {} seconds ago!", n.as_secs()),
    Err(_) => panic!("SystemTime before UNIX EPOCH!"),
}
```

Of course without using the `use` statement. Also move this to a utils or something. This gives `n` as Duration, use [`.as_nanos()`](https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_nanos) to get nanoseconds duration.

Reply:
For now lets panic if system time is reported as before 1970 something.

Reply:
Remove `chrono` dependency.